_sass/_fonts.scss

```diff
@@ -20,11 +64,55 @@
   font-weight: normal;
   line-height: 1;
   -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale
+  -moz-osx-font-smoothing: grayscale;
+
+  /* added by EITI developers */
+  vertical-align: text-bottom;
+}
+
+.icon-padded:before {
+  padding-right: $base-padding-lite;
+}
+
+.icon-gold:before{content:'\0041';}
+.icon-close-x:before{content:'\0042';}
+.icon-renewables:before{content:'\0043';}
+.icon-book:before{content:'\0044';}
+.icon-geo:before{content:'\0045';}
+[class*='icon-chevron-sm-']:before,
+.icon-chevron-sm:before {
+  content: '\0046';
 }
-[class*='icon-'] {
-  font-size: 1.5em;
-  vertical-align: middle;
-  margin-right: 5px;
+.icon-copper:before{content:'\0047';}
+.icon-map:before{content:'\0048';}
+.icon-dash-lg:before{content:'\0049';}
+.icon-dash-sm:before{content:'\004A';}
+.icon-cloud:before{content:'\004B';}
+.icon-plus-lg:before{content:'\004C';}
+.icon-minerals:before{content:'\004D';}
+.icon-oil:before{content:'\004E';}
+.icon-wind:before{content:'\004F';}
+.icon-coal:before{content:'\0050';}
+.icon-hardrock:before{content:'\0051';}
```

Comment:
Colon after property should be followed by one space<br>Opening curly brace `{` should be preceded by one space